### ISSUE TITLE
Localization language

### ISSUE THREAD

**Issue by EmperorAtlantis** [dotnet/core#2970]
# Issue Title

Localization language

# General

Can I have various localized language packages like the.net Framework? When I use vs, I want to see annotated descriptions of classes or methods in Chinese.

**Comment by karelz** [comment]
I don't think we have localized help. @mairaw do you know more or can you please route it?

**Comment by mairaw** [comment]
Not yet @karelz, but we're working on enabling that. Not sure if for 3.0 or a later release at this point.

/cc @ahsonkhan @safern @JasonCard @carlossanlop

**Comment by carlossanlop** [comment]
Should we tranfer this issue to the dotnet-api-docs repo, @mairaw?

**Comment by mairaw** [comment]
I think this belongs here because we need to check-in the generated IntelliSense files into the product but it's up to you.

**Comment by EmperorAtlantis** [comment]
Depends on me? Is there anything I can do for you?

**Comment by karelz** [comment]
@mairaw which repos does it affect? Just CoreFX? Or more?

**Comment by mairaw** [comment]
I don't know where the packages are created but I imagine it would also impact other repos like winforms, etc.

**Comment by carlossanlop** [comment]
Moving to the dotnet/dotnet-api-docs repo for better tracking.
